src/server/middlewares/validator.js

```diff
@@ -32,5 +32,47 @@ class Validate {
       DisplayError(err, res);
     }
   }
+
+  static createTodo (req, res, next){
+    const { title, description, completed } = req.body;
```

Comment:
I reviewed ```todoController.create()``` in  ```controllers/todo.js``` and found out that in your controller, you used ```const { userId } = req.params;```  but you didn't validate the param your ```createTodo()``` in ```validator.js```

Reply:
I've made the changes